azext_iot/_help.py

```diff
@@ -1080,6 +1080,71 @@
         az iot edge deployment show-metric -m myCustomMetric -d {deployment_name} -n {iothub_name}
 """
 
+helps[
+    "iot edge image"
+] = """
+    type: group
+    short-summary: Manage IoT Edge images.
```

Comment:
nit: the word module is fine to include in the help descriptions, just the command invocation was getting too long.